index.html

```diff
@@ -9,57 +9,246 @@
+                        <p><a href="#">JavaScript</a></p>     
+                    </div>
+                    <div class="btn-container">
+                        <div class="btn"><a href="#">See project</a></div>
+                    </div>
+                </div>
+            </section>
+        </section>
+
+        <!-- About Myself -->
+
+        <section class="aboutme">
+            <section class="aboutme-header">
+                <h1>About Myself</h1>
                 <p>
-                    I’m a software developer! I can help you build a product , feature
-                    or website Look through some of my work and experience! If you
-                    like what you see and have a project you need coded, don’t
-                    hestiate to contact me.
+                    Hello I’m a software developer! I can help you build a 
+                    product, feature or website. Look through some of my 
+                    work and experience! If you like what you see and have 
+                    a project you need coded, don’t hesitate to contact me.
                 </p>
-                <h3>LET’S CONNECT</h3>
-                <ul class="social-icons">
-                    <li>
-                        <a href="#"><img src="./assets/social/twitter.svg" alt="Twitter"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/Linkedin.svg" alt="Linkedin"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
-                    </li>
-                </ul>
-            </div>
+                <div class="letcon">
+                    <h3 class="let_connect">LET’S CONNECT</h3>
+                    <ul class="social-icons">
+                        <li>
+                            <a href="#"><img src="./assets/social/twitter.svg" alt="Twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/Linkedin.svg" alt="Linkedin"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
+                        </li>
+                    </ul>
```

Comment:
- [ ] Kindly pay attention to the little margin left to the list of the icons on the Figma design and apply the same on your design

### Your Design
![your-icon-space](https://user-images.githubusercontent.com/26597432/236419435-8c5fa6db-fdbb-4f5c-9c6c-e82295db16fc.png)

### Figma Design
![temp-icon-space](https://user-images.githubusercontent.com/26597432/236419476-9a5dbfb8-7eb0-473b-8e9b-18b97bb9c549.png)